File: src/main/java/com/bokeunjeong/portfolio/repository/PortfolioSkillRepository.java
package com.bokeunjeong.portfolio.repository;

import com.bokeunjeong.portfolio.model.Skill;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface PortfolioSkillRepository extends JpaRepository<Skill, String> {

    public List<Skill> findAllByDisplay(Boolean display);

}

File: src/main/kotlin/com/bokeunjeong/portfolio/repository/PortfolioSkillRepository.kt
package com.bokeunjeong.portfolio.repository

import com.bokeunjeong.portfolio.dto.PortfolioSkillDto
import org.springframework.data.jpa.repository.JpaRepository


interface PortfolioSkillRepository : JpaRepository<PortfolioSkillDto, Int> {
    fun findByNo(no: Int): List<PortfolioSkillDto>
    fun findByName(name: String): List<PortfolioSkillDto>
    fun findByType(type: String): List<PortfolioSkillDto>
    fun findByUseYn(useYn: String): List<PortfolioSkillDto>
}